ui/components/ui/tabs/snaps/dropdown-tab/dropdown-tab.test.js

```diff
@@ -36,7 +36,7 @@ describe('DropdownTab', () => {
   it('registers selection', () => {
     const { container, getByText } = render(<DropdownTab {...args} />);
 
-    fireEvent.click(container.firstChild);
+    fireEvent.click(container.firstChild.firstChild.lastChild);
```

Comment:
The PR looks good and great add-ons to migrate some components styles. 
question: if there a way to simplifier this selector? if not, might worth some explanations to make the code understandable ~

Reply:
I added the comment and made code a bit more self-explanatory, I hope that helps. I find this API for selecting these kind of elements very limited. Let me know if there is a better example that is not adding more code to the functional code, I'll take a look and implement if possible.